packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -200,8 +200,11 @@ export const mutation = mutationType({
 
     t.liveMutation('launchOpenProject', {
       description: 'Launches project from open_project global singleton',
+      args: {
+        specPath: stringArg(),
```

Comment:
Should this `nonNull(stringArg())`?

Reply:
No, since this mutation is used in other places that don't require routing to a specific spec.